Applications/DataExplorer/mainwindow.h

```diff
@@ -47,7 +49,6 @@ class MainWindow : public QMainWindow, public Ui_MainWindowClass
 
 public:
 	MainWindow(QWidget* parent = 0);
-	~MainWindow();
```

Comment:
Next time it would be really nice if you could wait with the rebasing until after code review, s.t. it is clearly visible which changes you made later on.